spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -75,6 +77,32 @@ module CCMS # rubocop:disable Metrics/ModuleLength
         end
       end
 
+      context 'FAMILY_PROSPECTS_OF_SUCCESS' do
+        it 'returns the ccms equivalent prospect of success for likely' do
+          allow(legal_aid_application.merits_assessment).to receive(:success_prospect).and_return('likely')
```

Comment:
So a mock is being stubbed to return a value in a spec .... so what, if anything, is being tested here?

Reply:
I was aiming to test that the prospects of success assigned by the Apply service were translated to the correct output, e.g. when Apply assigns 'likely' this is translated to the CCMS equivalent and populates the block with 'Good'

Reply:
that seems reasonable to me, or am I missing something?